codegen-test/model/misc.smithy

```diff
@@ -18,6 +18,7 @@ service MiscService {
         ResponseCodeRequiredOperation,
         ResponseCodeHttpFallbackOperation,
         ResponseCodeDefaultOperation,
+        AcceptHeaderStarService,
```

Comment:
Shouldn't this be added to `codegen-server-test`?

Reply:
It's just a symlink:
```
$ ls -l codegen-server-test/model/misc.smithy
codegen-server-test/model/misc.smithy -> ../../codegen-test/model/misc.smithy
```